x/blob/client/cli/testrandblob.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/celestiaorg/celestia-app/testutil/namespace"
+	"github.com/celestiaorg/celestia-app/testutil/testfactory"
+	"github.com/celestiaorg/celestia-app/x/blob/types"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+)
+
+// CmdTestRandBlob is triggered by testground's tests as part of apps' node scenario
+// to increase the block size by user-defined amount.
+//
+// CAUTION: This func should not be used in production env!
+func CmdTestRandBlob() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "TestRandBlob [blobSize]",
+		Short: "Generates a random blob for a random namespace to be published to the Celestia blockchain",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// decode the message size
```

Comment:
[nit] sorry not sure how I missed this on initial review
```suggestion
			// decode the blob size
```